CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Default to Python 3.10
 - Default to Elasticsearch 8.x
+- Collection objects are not stored in `collections` rather than `stac_collections`
```

Comment:
This is confusing, can you make this an affirmative statement (e.g. collection objects *are* stored in ... instead of ...).

Reply:
Collection objects are now stored ...

Reply:
now vs. not - spelling error?

Reply:
spelling error, corrected.